components/script/dom/filelist.rs

```diff
@@ -18,6 +18,7 @@ pub struct FileList {
 }
 
 impl FileList {
+    #[allow(unrooted_must_root)]
     fn new_inherited(files: Vec<JS<File>>) -> FileList {
```

Comment:
Anything named `new` or `new_something` is supposed to get automatically white-listed, weird that you had to add these.